examples/erc1155/src/lib.rs

```diff
@@ -47,46 +50,69 @@ impl Erc1155Example {
         amounts: Vec<U256>,
         data: Bytes,
     ) -> Result<(), Vec<u8>> {
+        self.pausable.when_not_paused()?;
         self.erc1155._mint_batch(to, token_ids, amounts, &data)?;
         Ok(())
     }
 
-    pub fn set_operator_approvals(
```

Comment:
Forgot to remove this function in one of previous PRs, my bad

Reply:
Np at all!